views/index.html

```diff
@@ -188,23 +223,24 @@ <h3 class="username-block">
       
       <div class="container">
        <div class="row">
-         <div class="col-6s">
```

Comment:
I think the extra s here was a typo. Using bootstrap it should be a col-6 maybe. It looks funky on mobile but it will get me through the demo at least.